src/pages/android-development/index.md

```diff
@@ -0,0 +1,16 @@
+# Android Development
```

Comment:
Add before the first line:
```
---
title: Android Development
---
```

Reply:
Hi @bla-ke , I actually submitted a new PR with this change earlier today and that passed the checks. I will close this PR.

Thanks!